prepare/cards/sst2.py

```diff
@@ -29,9 +29,7 @@
 add_to_catalog(one_sentence_classification_templates, "templates.one_sent_classification", overwrite=True)
 
 one_sentence_classification_task = FormTask(
-    inputs=["choices", "sentence"],
-    outputs=["label"],
-    metrics=["metrics.accuracy"],
+    inputs=["choices", "sentence"], outputs=["label"], metrics=["metrics.accuracy"], augmentable_inputs=["sentence"]
```

Comment:
what is the augmentation?

Reply:
There are two modes of augmentation.    One where the entire input provide to the model is augmented (called model input augmentation) .  This includes the any system prompt,  instructions or demonstrations.   The second type of augmentation (called task input augmentation) , augments only specific fields in the task's input.   Augmentable inputs are the subsets of the task's input field that undergo  augmentation.  For example, in sst ,  only the 'sentence' input is augmentwed and not the "choices", which is fixed.

Reply:
yes, but is it whitespace augmentation? where is it stated?